docs/UserGuide.md

```diff
@@ -17,29 +17,60 @@ Source Control is **optimized for use via a Command Line Interface** (CLI) while
 If you can type fast, Source Control can help you track your students' performance faster than traditional GUI apps!
 
 This guide takes you through all the _latest features_ of Source Control. If you are a new user, this guide provides you
-all the basic knowledge to get started with Source Control. If you are an experienced user,
-feel free to skip to the [command summary](#command-summary) and experiment with our more advanced features!
+all the basic knowledge to [get started](#quick-start) with Source Control. If you are an experienced user,
+feel free to skip to the [command summary](#command-summary) and experiment with our more [advanced features](#setting-customised-aliases-for-commands--alias)!
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Table of Contents
 
-* Table of Contents
-{:toc}
+* [Overview](#)
+* [Table of Contents](#table-of-contents)
+* [Quick Start](#quick-start)
+* Features
+  * [Notes on command format](#features)
+  * Adding data
+    * [Adding a student : `add student`](#adding-a-student--addstudent)
+    * [Creating a new group : `add group`](#creating-a-new-group--addgroup)
+    * [Adding a student into a group : `add alloc`](#adding-a-student-into-a-group--addalloc)
+    * [Creating a new assessment : `add assessment`](#creating-a-new-assessment--addassessment)
+    * [Adding a score : `add score`](#adding-a-score--addscore)
+  * Searching Data
+    * [Listing all students : `list`](#listing-all-students--list)
+    * [Searching for students : `search`](#searching-for-students--search)
+  * Analysing Data
+    * [Showing assessment result analysis : `show`](#showing-assessment-result-analysis--show)
+  * Editing Data
+    * [Editing a student : `edit`](#editing-a-student--edit)
+    * [Deleting a student : `delete`](#deleting-a-student--delete)
+  * Import and Export
+    * [Importing data : `import`](#importing-data--import)
+    * [Exporting data : `export`](#exporting-data--export)
+  * General features
+    * [Resetting all data : `clear`](#resetting-all-data--clear)
+    * [Closing the app : `exit`](#closing-the-app--exit)
+    * [Viewing Help : `help`](#viewing-help--help)
+    * [Accessing command history](#accessing-command-history)
+  * Advanced Features
+    * [Setting customised aliases for commands: `alias`](#setting-customised-aliases-for-commands--alias)
```

Comment:
ive searched through the document everything is written as "se" ;-; lets use British spelling please

Reply:
Ohh okay! I thought we were using American style since I notice this first haha (but it seems like it is the only one...), so yea may I trouble u to change it from "optimized" to "optimised" >;<

![image](https://user-images.githubusercontent.com/63232300/139900927-95633ff6-46a2-486d-9455-4bebf6086dbf.png)